algorithms/kernel/linear_model/linear_model_train_normeq_update_impl.i

```diff
@@ -87,12 +88,16 @@ Status ThreadingTask<algorithmFPType, cpu>::update(DAAL_INT startRow, DAAL_INT n
     DAAL_CHECK_BLOCK_STATUS(_yBlock);
     const algorithmFPType *y = _yBlock.get();
 
-    Blas<algorithmFPType, cpu>::xxsyrk(&up, &notrans, &nFeatures, &nRows, &alpha,
-                                       const_cast<algorithmFPType *>(x), &nFeatures, &alpha,
-                                       _xtx, &_nBetasIntercept);
+    {
+        DAAL_ITTNOTIFY_SCOPED_TASK(computeUpdate.syrkX);
+        Blas<algorithmFPType, cpu>::xxsyrk(&up, &notrans, &nFeatures, &nRows, &alpha,
+                                        const_cast<algorithmFPType *>(x), &nFeatures, &alpha,
+                                        _xtx, &_nBetasIntercept);
+    }
 
     if (nFeatures < _nBetasIntercept)
     {
+        DAAL_ITTNOTIFY_SCOPED_TASK(computeUpdate.gemm1X);
```

Comment:
Is this really a gemm stage?

Reply:
Yes, it is equivalent of gemm with vector of ones from GPU version of implementation